framework/src/binary_manager/binary_manager_internal.h

```diff
@@ -25,6 +25,18 @@
 
 #include <tinyara/binary_manager.h>
 
+#define SEND_REQUEST(type)								\
+	do {										\
+		request_msg.cmd = (type);						\
+		request_msg.requester_pid = getpid();					\
+		snprintf(request_msg.data.bin_name, BIN_NAME_MAX, "%s", binary_name);	\
+		ret = binary_manager_send_request(&request_msg);			\
```

Comment:
header for binary_manager_send_request

Reply:
check again

Reply:
binary_manager_send_request is defined in this file. Therefore, there is no need to include header.